templates/consent1.html

```diff
@@ -35,25 +36,29 @@
 			if (txtPanel && txtPanel.clientHeight > 0) {
 				txtPanel.style.height = "0px"
 				txtPanel.style.transition = "height 0.0s"
-				txtPanel.style.height = txtPanel.scrollHeight + "px"
```

Comment:
Wouldn't this make it so reszing acts like it closes everything instead of making everything automatically re-adjust? Did this not work on your computer or am I missing something?

Reply:
Yeah -- I can confirm this introduces a regression for the bug I fixed earlier.